### PR TITLE
added dice loss file

### PR DESCRIPTION
**Pull request by gamenerd457**
# In this commit i have added dice loss to losses folder

> * dice loss can be used as a loss function for training segmentation . models
>

## Tests

tested the file using unittest

### CODE REVIEW

**Comment by saberkun** [comment]
keras_cv is ramping down in this repo now. We do not accept changes to this folder. Thanks
